app/build.gradle

```diff
@@ -11,17 +11,16 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
 
 def apiKey = getValueFromFileOrEnv(APP_TOKEN_ENV_VAR_NAME, APP_TOKEN_FILE, APP_TOKEN_KEY, ERROR_MESSAGE)
 
-def sdkVersion = '2.4.0'
+def sdkVersion = '3.0.0'
 def versionCodeNumber = 11
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.3'
+    compileSdkVersion 27
 
     defaultConfig {
         applicationId "com.onfido.sampleapp"
         minSdkVersion 16
-        targetSdkVersion 25
+        targetSdkVersion 27
```

Comment:
any reason why we are upgrading the target? just to be up to date?

Reply:
to be up to date + when the bump dependencies PR go in, we'll need to do so anyway (required by gradle 3.1.0)

Reply:
sorry actually the bump-dependencies PR has nothing to do with it, it's required by the bump of gradle on **this PR**